src/main/java/tech/ydb/spark/connector/YdbTable.java

```diff
@@ -18,23 +18,22 @@
 import org.apache.spark.sql.connector.expressions.Expressions;
 import org.apache.spark.sql.connector.expressions.Transform;
 import org.apache.spark.sql.connector.read.ScanBuilder;
-import org.apache.spark.sql.connector.write.LogicalWriteInfo;
-import org.apache.spark.sql.connector.write.RowLevelOperationBuilder;
-import org.apache.spark.sql.connector.write.RowLevelOperationInfo;
-import org.apache.spark.sql.connector.write.WriteBuilder;
+import org.apache.spark.sql.connector.write.*;
```

Comment:
YDB code style does not allow wildcard imports

Reply:
just noticed there was checkstyle config in the repo, but it seems to be alright with wildcard imports (tried both idea plugin and maven run)